packages/legacy/core/App/screens/ProofRequest.tsx

```diff
@@ -255,6 +257,22 @@ const ProofRequest: React.FC<ProofRequestProps> = ({ navigation, route }) => {
     ...retrievedCredentials?.predicates,
   })
 
+  useEffect(() => {
+    // get oca bundle to see if we're presenting personally identifiable elements
+    proofItems.some(async (item) => {
```

Comment:
Missed this on my first readthrough, what is the purpose of the first `some` here? It doesn't seem to be used anywhere

Reply:
Could probably just be a `forEach`

Reply:
I'm checking to see if any of the proof items contains a flagged attribute. Some exits after the first success so it's a bit faster/more relevant than a foreach

Reply:
Oh gotcha. Nice one, approved